### ISSUE TITLE
Dependency conflict with pytest and selenium

### ISSUE THREAD

**Issue by PhilippSelenium** [pytest-dev/pytest-selenium#294]
```
pytest-selenium 3.0.0 requires pytest<7.0.0,>=6.0.0, but you have pytest 7.1.1 which is incompatible.
pytest-selenium 3.0.0 requires selenium<4.0.0,>=3.0.0, but you have selenium 4.1.2 which is incompatible
```

**Comment by PhilippSelenium** [comment]
Just saw this MR https://github.com/pytest-dev/pytest-selenium/pull/293 ...

**Comment by BeyondEvil** [comment]
Yes, the idea is that v3.* will be for Selenium 3 support (which we'll keep around another year or so), while v4.* will be for Selenium 4 support.

**Comment by icemac** [comment]
@BeyondEvil Is there a roadmap for releasing version 4.x? Currently I am using 2.x against selenium 4.x which still works fine (besides some deprecation warnings). Not being able to use the next major version of pytest-selenium because it no longer supports the current selenium version seems a bit strange.

**Comment by BeyondEvil** [comment]
> @BeyondEvil Is there a roadmap for releasing version 4.x? Currently I am using 2.x against selenium 4.x which still works fine (besides some deprecation warnings). Not being able to use the next major version of pytest-selenium because it no longer supports the current selenium version seems a bit strange.

As soon as #272 is merged, I plan on releasing `v4.0.0`.

Not sure what you mean, but there will be two parallel tracks - v3* that will continue to support Selenium 3 and v4* that will support Selenium 4.

Would that cause an issue for you?

**Comment by icemac** [comment]
@BeyondEvil I see no problem in having the two parallel tracks. I just wondered that 2.x works fine with selenium 4.x while 3.x dropped the support and I have to wait for 4.0 to get it back again.

I am not in a hurry: not using pytest-selenium 3.x is fine for me.

**Comment by icemac** [comment]
Hi, are there any news on the planned 4.0 release of this package?

**Comment by BeyondEvil** [comment]
> Hi, are there any news on the planned 4.0 release of this package?

Hi,

Sorry, not at this moment.

We're working hard on a new release (rebuild from scratch) of pytest-html currently.

As soon as that has been released, then we'll make sure pytest-selenium gets a refresh as well (with full sel 4.0 support).

Thank you for your patience! 🙏

**Comment by BeyondEvil** [comment]
v4.0.0 is released!

Thank you for your patience! @icemac

**Comment by icemac** [comment]
You are awesome, thank you!
